src/toolbox/ngm-point-edit.js

```diff
@@ -113,6 +127,30 @@ class NgmPointPosition extends I18nMixin(LitElement) {
                     class="ngm-height-input" .value="${this.heightValue}" @change="${this.onPositionChange}">
                 <label class="ui label">m</label>
             </div>
+            <button class="ui icon button ngm-aoi-point-style-btn" style="margin-left: 84px;margin-right: 0;">
```

Comment:
Can the style be put in the CSS file?

Reply:
I missed it during implementation, moved to css